patientIdentifierType.md

```diff
@@ -0,0 +1,112 @@
+# PatientIdentifierType
+
+## Overview
+Administrators define what types of identifiers they will collect. These range from National ID numbers, to driver's license numbers, to per-hospital medical record numbers.
+
+### Properties on PatientIdentifierType:
+
+* format: a regular expression defining what the identifier text should contain.
+
+* formatDescription: An optional description of the regular expression (used to explain the requirements of the regular expression in terms a user would understand). For example, a regular expression like \d{4,8}could have a description like "Must be a number between 4 and 8 digits in length."
+
+* required: a true/false whether every patient MUST have this type
+
+* checkDigit: a true/false whether this identifier has a checkdigit at the end.
+
+* validator: full class name of the IdentifierValidators (for example : `org.openmrs.patient.IdentifierValidator`).
```

Comment:
[org.openmrs.patient.IdentifierValidator](https://docs.openmrs.org/doc/org/openmrs/patient/IdentifierValidator.html) is not an example of a validator; rather, it's _the_ interface that must be implemented by a validator. An example would the [LuhnIdentifierValidator](https://docs.openmrs.org/doc/org/openmrs/patient/impl/LuhnIdentifierValidator.html). So, the description could be:

```
* validator: full class name of an [IdentifierValidator](https://docs.openmrs.org/doc/org/openmrs/patient/IdentifierValidator.html) (e.g., [`org.openmrs.patient.impl.LuhnIdentifierValidator`](https://docs.openmrs.org/doc/org/openmrs/patient/impl/LuhnIdentifierValidator.html))
```

Reply:
sure, wll update